config/app.js

```diff
@@ -1,5 +1,6 @@
 export default {
   name: 'Apollo Starter Kit',
+  logo: 'logo.svg', // Loaded via webpack and stored in favicon/common/assets
```

Comment:
I don't see where does this property used in this PR?

Reply:
It isn’t used, it’s optional for someone to use it elsewhere. I put it in the docs.